lib/constants/Enumerations.h

```diff
@@ -59,7 +60,7 @@ namespace BuildingSubID
 enum class EMarketMode : int8_t
 {
 	RESOURCE_RESOURCE, RESOURCE_PLAYER, CREATURE_RESOURCE, RESOURCE_ARTIFACT,
-	ARTIFACT_RESOURCE, ARTIFACT_EXP, CREATURE_EXP, CREATURE_UNDEAD, RESOURCE_SKILL,
+	ARTIFACT_RESOURCE, ARTIFACT_EXP, CREATURE_EXP, ARTIFACT_CREATURE_EXP, CREATURE_UNDEAD, RESOURCE_SKILL,
```

Comment:
Do we really need once more market mode?

Reply:
No, but should be the cleanest way, I think?

Alternatives?